nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -144,6 +144,9 @@ NessieCatalog initCatalog(String ref, String hash) {
             .put(CatalogProperties.URI, uri)
             .put("auth-type", "NONE")
             .put(CatalogProperties.WAREHOUSE_LOCATION, temp.toUri().toString());
+    if (api instanceof NessieApiV2) {
```

Comment:
We should probably make the API version available via injected test parameters (needs a Nessie change), for example in `NessieClientFactory`

Reply:
API version is explicitly accessible now, cf. https://github.com/projectnessie/nessie/blob/9558d8e105a1e33763918c0336d8020118e4f7e3/servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/BaseTestNessieApi.java#L131

Reply:
oh nice. Updated it.